src/state/migrations/__test__/3.test.ts

```diff
@@ -0,0 +1,63 @@
+import { afterEach, describe, expect, it } from 'vitest';
+
+import { DisplayUnit } from '@/constants/trade';
+
+import {
+  AppColorMode,
+  AppTheme,
+  initialState as appUiConfigsInitialState,
+} from '@/state/appUiConfigs';
+
+import { V2State } from '../2';
+import { AppUiConfigsKeys, appUiConfigsLocalStorageKeys, migration3 } from '../3';
+
+const V2_STATE: V2State = {
+  _persist: { version: 2, rehydrated: true },
+  dismissable: {
+    hasSeenPredictionMarketIntroDialog: false,
+    dismissedAffiliateBanner: false,
+  },
+};
+
+const oppositeUiConfigsLocalStorageValues = {
+  appThemeSetting: AppTheme.Light,
+  appColorMode: AppColorMode.RedUp,
+  hasSeenLaunchIncentives: true,
+  defaultToAllMarketsInPositionsOrdersFills: false,
+  displayUnit: DisplayUnit.Fiat,
+  shouldHideLaunchableMarkets: true,
```

Comment:
default should be `false`

Reply:
unless this is part of the test

Reply:
part of the test! this is the opposite values